lib/utils/FileCache.mjs

```diff
@@ -59,6 +59,7 @@ export default class FileCache {
 		// Get file descriptor compares current metadata against cached
 		// one and stores the result to "changed" prop.w
 
+		const metaCache = this._fileCache.cache.getKey(this._fileCache.createFileKey(absoluteFilepath));
```

Comment:
[suggestion] I see `metaCache` is an `any` type in my IDE, so how about typing the variable like this?

```suggestion
		/** @type {import('file-entry-cache').FileDescriptorMeta | undefined} */
		const metaCache = this._fileCache.cache.getKey(this._fileCache.createFileKey(absoluteFilepath));
```